js/exercise_one/index.js

```diff
@@ -0,0 +1,90 @@
+function sortTableByCost(table) {
+
+  const tBody = table.tBodies[0];
+  const rows = Array.from(tBody.querySelectorAll("tr"));
+
+  const sortedRows = rows.sort((a, b) => {
+    const aCost = parseFloat(a.querySelector(`td:nth-child(${ 1 + 1})`).textContent.substring(1));
+    const bCost = parseFloat(b.querySelector(`td:nth-child(${ 1 + 1})`).textContent.substring(1));
+    return aCost >= bCost ? 1 : -1;
+  });
+
+  while(tBody.firstChild){ // ACA SE TRANCA
+    tBody.removeChild(tBody.firstChild);
+  }
+  tBody.append(...sortedRows);
+}
```

Comment:
why are you doing this?

Reply:
Otherwise it may fail in some situations. It expands an iterable into more elements. sortedRows has many rows, i think you can not add it without ...